docs/content/tutorial-cmake.md

```diff
@@ -87,6 +88,20 @@ should not be included in the installed version of the library. This prevents
 details of your build from leaking into the install. We'll handle the include
 directory for the installed library differently.
 
+There is one last step for creating the C library. Remember how we passed
```

Comment:
I suggest moving this to a separate section labeled something like "Advanced usage" or "Generating types for use in shared libraries". I suspect most users will not be creating shared libraries of types on their first pass, and introducing it so early will end up being confusing.